Index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="header-class"> "Hello Microverse!"</div>
```

Comment:
- [ ] **OPTIONAL** Please remove the extra space before the "Hello Microverse!" for more consistency.
It's always better to have completely clean code files. It helps maintain professionally and accurately.